src/main/java/com/webonise/rbs/constants/RedirectStatus.java

```diff
@@ -0,0 +1,19 @@
+package com.webonise.rbs.constants;
+
+public enum RedirectStatus {
+
+    success("success"),
```

Comment:
Make it in uppercase, Enum member are constant please follow the convention of constant.